manager/manager.go

```diff
@@ -108,6 +121,7 @@ type Manager struct {
 	server                 *grpc.Server
 	localserver            *grpc.Server
 	raftNode               *raft.Node
+	keyRotator             *RaftDEKManager
```

Comment:
Let's call it `kekRotator`. There's already something called `keyManager` that does key rotation. Arguably we should give `keyManager` a more specific name as well.